### DISCUSSION TITLE
Plans to Add Next-Auth?

### DISCUSSION THREAD

**Discussion by kissngg** [BearStudio/start-ui-web#442] [Q&A] [answered]
I'm looking to implement social logins easily using next-auth, and I noticed in the commit history that there was an attempt to introduce next-auth but it was removed. Are there any plans for this in the future?

**Comment by ivan-dalmet** [comment, marked as answer]
Hello @kissngg 
We stopped to use Next Auth because, out of the box, it's only working in a browser context (with cookies) and we have a lot of mobile apps (with React Native).
We still want, in a long run, to use Next Auth and make sure we can use it outside of the browser but it's will take time.
In a shorter run we want to provide a guide in the documentation to use Next Auth (if you don't need mobile app support).
But you know, time is a limited resource 😅

**Comment by ivan-dalmet** [comment]
(I will convert this issue into a discussion 😉)
